### ISSUE TITLE
Types missing when using in typescript module?

### ISSUE THREAD

**Issue by sharunkumar** [davidnguyen11/p-queue-ts#24]
I am getting the following error on import:

![image](https://github.com/davidnguyen11/p-queue-ts/assets/715417/5b95f493-b942-4a70-941c-9175dc175b27)

trying to add the types through yarn also gives an error:
```ps1
➜ yarn add --dev @types/p-queue-ts
yarn add v1.22.19
[1/4] Resolving packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@types%2fp-queue-ts: Not found".
info If you think this is a bug, please open a bug report with the information provided in "E:\\Code\\AdventOfCode\\typescript\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

for more context, I am trying to use this to solve advent of code, code can be found here:
https://github.com/sharunkumar/AdventOfCode/blob/pqueue-error/typescript/src/solutions/day-12-hill-climbing-algorithm-1.ts

**Comment by sharunkumar** [comment]
Figured out the cause of this, I have `"strict": true,` in my TSConfig, which is why this issue is occuring. though, shouldn't this probably be addressed in the lib as well? 🤔

**Comment by sharunkumar** [comment]
I've fixed this in #25 and have tested it against my repo which is in strict mode: https://github.com/sharunkumar/AdventOfCode/blob/day-12/typescript/src/solutions/day-12-hill-climbing-algorithm-1.ts

no error now

**Comment by davidnguyen11** [comment]
I merged your PR  and published new version. Thank you for contributing!

https://github.com/davidnguyen11/p-queue-ts/releases/tag/1.2.0
